cocos/editor-support/cocosbuilder/CCBAnimationManager.cpp

```diff
@@ -99,7 +99,11 @@ Node* CCBAnimationManager::getRootNode()
 
 void CCBAnimationManager::setRootNode(Node *pRootNode)
 {
+    CC_SAFE_RELEASE_NULL(_rootNode);
+    
     _rootNode = pRootNode;
+    
+    CC_SAFE_RETAIN(_rootNode);
```

Comment:
@HoGarfield  So could you add it in this PR?
 After it CI is ready, we could merge this patch.
Thanks.